glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -548,12 +547,11 @@ additionalProperties:
             type:
               enum:
                 - custom_distribution
-                - quantity
         then:
           required:
             - gecko_datapoint
           description: |
-            `custom_distribution` and `quantity` is only allowed for Gecko
+            `custom_distribution` is only allowed for Gecko
```

Comment:
Did you check if you're allowed to use `quantity` without a `gecko_datapoint` property?

Reply:
Manually yes, do you think I should add a unit test?

Reply:
Nope, that's fine. If you checked manually and it works, I trust you :)

Reply:
Also the `test_quantity` test kinda implicity tests this since it now only gets one error and not two. Which reminds I should update it's docstring.